### DISCUSSION TITLE
Is Rallly working without an SMTP server?

### DISCUSSION THREAD

**Discussion by albertopasqualetto** [lukevella/rallly#648] [Q&A] [answered]
Is Rallly working without an SMTP server?

**Comment by lukevella** [comment]
Technically, it will run. You need to leave out SUPPORT_EMAIL I believe. Registration, login and notifications won’t work of course.

**Reply by albertopasqualetto** [reply]
Good, so, to work without a server it would be good to have an admin login directly in the .env file.

**Comment by PLP-GTR** [comment, marked as answer]
Yes, it works. 👍

Leave out `SUPPORT_EMAIL` in `config.env` which skips sending, see logs:
```log
rallly-app  | SUPPORT_EMAIL not configured - skipping email send
```

If you misconfigure the SMTP it will still work but log that an error occured while it tried to send an email:
```log
rallly-app  | Error sending email
```
